src/Symfony/Component/Cache/composer.json

```diff
@@ -26,6 +26,7 @@
     "require-dev": {
         "cache/integration-tests": "dev-master",
         "doctrine/cache": "~1.6",
+        "doctrine/dbal": "~2.4",
```

Comment:
From a "component" point of view, you may add ``doctrine/dbal`` to the ``suggest`` section, what do you think ?